### ISSUE TITLE
Create new release

### ISSUE THREAD

**Issue by wtimme** [holyturt/SwiftOverpassWrapper#18]
With all the recent changes, we should release a new version.

## Version Number

We've updated the models and the queries, which are breaking API changes for clients. If we were to follow [Semantic Versioning 2.0.0][1], the new release would be `2.0.0`. What do you think, @holyturt?

## Tasks
- Update `spec.version` in `SwiftOverpass.podspec` and commit
- Tag that commit

[1]: https://semver.org/

**Comment by holyturt** [comment]
Yes, It seems it's time to release a new version.
Do you have any WIP branches you want to merge before the new version? If you don't I will update to the new version.

**Comment by wtimme** [comment]
Glad to hear! No, at the moment, I don't have any branches that would prevent a release, so you can go right ahead. Thanks in advance!

**Comment by holyturt** [comment]
It's done. Thank you for the tasks. It was helpful. Furthermore, thank you for your great contributions, @wtimme!

**Comment by wtimme** [comment]
Hey @holyturt,

Thanks for your effort! Unfortunately, I have issues installing that version using Cocoapods:
```
[!] CocoaPods could not find compatible versions for pod "SwiftOverpass":
  In Podfile:
    OSMSwift (from `../`) was resolved to 0.2.0, which depends on
      SwiftOverpass (~> 2.0.0)

None of your spec sources contain a spec satisfying the dependency: `SwiftOverpass (~> 2.0.0)`.
```
After some research online, I think that in order for a version to become available to use with a podspec, we need to add the new version to the Cocoapods "Trunk". It looks like the owner of the repo needs to add maintainers to Trunk in order for them to be able to push a new version.

What is required for us to release a new version with Cocoapods? Let's get the steps written down and documented. Looking forward to your feedback!
